Community/News/SCMS-is-now-a-chaoss-metric

```diff
@@ -0,0 +1,54 @@
+
+The numbers, trends, and other metrics CHAOSS offers provide a way to infer the community's status and where a community is going, but they are guessing at feelings, opinions, and voices that are clearly better found in qualitative data.
+
+The obvious and typical answer is to implement periodic surveys and pulse checks, but as we've stated in our blog on the 3 pitfalls of surveys, qualitative data is hard. Collecting it is a big part of the battle.
+
+It's not enough that it's difficult and time consuming to collect.  Once you have it, it's hard to know what's important, whose voices are silenced, and where the bias is. And then there’s sifting through it, finding reliable objective results in it, and knowing what to do with the findings.
+
+The SCMS approaches this problem directly.  
+
+In our view here at SC.O, virtual communities are inherently socio-cultural.  If communities are social in nature, they should be measured and studied social-scientifically.
+
+So we built the SCMS to simplify the same processes anthropologists use to study cultures, and created an interface that makes it easier and faster for business leaders to get qualitative results out of passive-user-sentiment. 
+
+At the heart of a community’s value, it’s not about what people have done, but how community members express themselves in the process of doing.
+
+People participate in communities for intrinsic reasons and those reasons are typically stated in their wants, opinions, and feelings as valuable, capturable data.
+
+Unfortunately it’s also "said in the moment" as it’s happening.  It’s fleeting. You won’t see it in a once-quarterly survey or focus group no matter how many gift cards you set up.  ​In other words, there’s value in measuring how people think and how they communicate in the moment.
+To do this The Social Currency Metrics System augments your quantitative data using a framework based on the theory of Social Currency, to measure moments your community exhibits trust in you.
+
+If you're interested in learning more on the system and setting it up for free let's get to next steps.  Here's how it works in Grimoire Labs.
+"At the heart of a community’s value, it’s not about what people have done, but how community members express themselves in the process of doing."
+
+​​~ Samantha Venia Logan
+WHAT'S NEXT FOR US, AND FOR YOU?
+There have been some amazing and wonderous developments before we've even been able to launch SociallyConstructed.Online's brand.  But now, with so much momentum, we've got a lot coming down the pipeline: 
+
+1st, remember that Software developer, Ria Gupta, you heard about?
+Keep a watch on this blog and subscribe for future releases because she is going to write a blog every sunday detailing how she's building the SCMS in Grimoire lab on here, her personal blog, and on the CHAOSS Blog!
```

Comment:
```suggestion
Keep a watch on this blog and subscribe for future releases because she is going to write a blog every Sunday detailing how she's building the SCMS in Grimoire lab on here, her personal blog, and on the CHAOSS Blog!
```